int8 robot1_ready
int8 robot2_ready
